packages/fuels-core/src/code_gen/custom_types_gen.rs

```diff
@@ -574,11 +568,11 @@ mod tests {
     fn test_expand_custom_struct_with_struct() {
         let p = Property {
             name: String::from("unused"),
-            type_field: String::from("struct cocktail"),
+            type_field: String::from("struct Cocktail"),
```

Comment:
Did you rename all of the type fields to match the expected ClassCase types for the tests, now that there is no longer a conversion to ClassCase?

Reply:
Yup, naming these structs just as you would when writing Sway. I could keep it as `cocktail` but then the expected should then be `cocktail` and not `Cocktail`, as we're not mutating it anymore.